VideoWeb/VideoWeb/ClientApp/src/app/services/conference/video-control-cache.service.ts

```diff
@@ -134,4 +185,12 @@ export class VideoControlCacheService {
         });
         return this.hearingControlStates?.participantStates[participantId]?.isLocalVideoMuted ?? false;
     }
+
+    private savingHearingState(syncChanges: boolean) {
+        if (syncChanges) {
```

Comment:
Would it make more sense to have the if check outside the function rather than passing in the boolean?

Reply:
will do